llvm/tools/llvm-mc/Disassembler.cpp

```diff
@@ -32,24 +34,25 @@ using namespace llvm;
 typedef std::pair<std::vector<unsigned char>, std::vector<const char *>>
     ByteArrayTy;
 
-static bool PrintInsts(const MCDisassembler &DisAsm, const ByteArrayTy &Bytes,
+static bool printInsts(const MCDisassembler &DisAsm, const ByteArrayTy &Bytes,
```

Comment:
is it possible to avoid unrelated refactoring? ditto for other places like skipToToken and the Disassemble variable in llvm-mc.cpp

Reply:
Done